### ISSUE TITLE
Feature Pyramid Networks

### ISSUE THREAD

**Issue by JihongJu** [broadinstitute/keras-rcnn#45]
FPN highlights the importance of leveraging the multi-scale feature hierarchy for object detection.

More details see the paper: [Feature Pyramid Networks for Object Detection](http://openaccess.thecvf.com/content_cvpr_2017/papers/Lin_Feature_Pyramid_Networks_CVPR_2017_paper.pdf)

**Comment by jhung0** [comment]
Yes, definitely want to implement this! We'll see the authors at cvpr in a
few days if you want us to ask any questions.

**Comment by JihongJu** [comment]
@jhung0 Ha, thank you for asking. I don't think I have any questions in particular for them atm. Enjoy! ;D

**Comment by hgaiser** [comment]
Bit late too @JihongJu  :p
